pandas/core/window.py

```diff
@@ -899,7 +899,53 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""
+    Calculate unbiased %(name)s kurtosis.
+
+    This function uses Fisher's definition of kurtosis without bias.
+
+    Parameters
+    ----------
+    kwargs : Under Review
+
+    Returns
+    -------
+    Series or DataFrame (matches input)
+        Like-indexed object containing the result of function application
+
+    See Also
+    --------
+    pandas.Series.%(name)s
+    pandas.DataFrame.%(name)s
+    pandas.Series.kurtosis
+    pandas.DataFrame.kurtosis
```

Comment:
The substituted name in this case is either `rolling` or `expanding` - does that answer your question?

Reply:
Ah, yes, that clarifies. 
Another point, to avoid that the See also section becomes long, we could also put the related series/dataframe versions on a single line (like `Series.kurtosis, DataFrame.kurtosis`)

Reply:
When you are worried about it becoming to long is that purely within the source or on the web as well? I believe that the HTML output styles this to display it all inline

Reply:
Ah, didn't know the html output put them on a single line anyhow. Not ideal IMO, but that's numpydoc behaviour.